bootstrap/util/configowner.go

```diff
@@ -42,7 +42,14 @@ type ConfigOwner struct {
 
 // IsInfrastructureReady extracts infrastructure status from the config owner.
 func (co ConfigOwner) IsInfrastructureReady() bool {
```

Comment:
Should we rename this?

```suggestion
func (co ConfigOwner) IsInfrastructureProvisioned() bool {
```

Reply:
Makes sense, but if this is ok for you I will do this in the follow up PR for Machine pools (it touches the same file)